controllers/provider-alicloud/charts/internal/shoot-controlplane/charts/csi-alicloud/templates/crd-csidriver.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if semverCompare "< 1.14" .Values.kubernetesVersion }}
```

Comment:
Can we rename the directory to `shoot-system-components` or something similar? I first got confused where these charts get applied.

Reply:
Done